library/accessory_workflows/build_cellranger_reference/BuildCellRangerReferenceBundleGencode.wdl

```diff
@@ -0,0 +1,123 @@
+version 1.0
+
+
+workflow BuildCellRangerRef {
```

Comment:
Could this workflow name and the task name below be distinguished from the ensembl version of this workflow? The wdl filename here makes it clear that it's Gencode but the workflows and tasks are then named the same. Maybe `BuildCellRangerGencodeRef`